week2-assignment/index.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+  <meta charset="UTF-8">
+  <link rel="icon" type="png" href="assets/logo.png" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css" />
+  <script src="https://kit.fontawesome.com/9c71be1e79.js" crossorigin="anonymous"></script>
+  <title>회원 관리 페이지</title>
+</head>
+<body>
+  <header>
+    <ul>
+      <li><i class="fa-solid fa-cat"></i></li>
+      <li><h1>회원 관리 페이지</h1></li>
+      <li><p>🌊AND SOPT 물결 웹 파이팅🌊</p></li>
+    </ul>
+  </header>
+  <main>
+    <section class="filters">
+      <div class="filter-group">
+        <div class="filter-item">
+          <label for="name">이름</label>
+          <input type="text" id="name" placeholder="이름을 입력하세요">
+        </div>
+        
+        <div class="filter-item">
+          <label for="eng-name">영문 이름</label>
+          <input type="text" id="eng-name" placeholder="영문 이름을 입력하세요">
+        </div>
+    
+        <div class="filter-item">
+          <label for="github">GitHub</label>
+          <input type="text" id="github" placeholder="GitHub ID를 입력하세요">
+        </div>
+
+        <div class="filter-item">
+          <label for="gender">성별</label>
+          <select id="gender">
+            <option value="" hidden selected>성별 선택</option>
+            <option value="male">남성</option>
+            <option value="female">여성</option>
+          </select>
+        </div>
+
+        <div class="filter-item">
+          <label for="role">역할</label>
+          <select id="role">
+            <option value="" hidden selected>OB/YB 선택</option>
+            <option value="OB">OB</option>
+            <option value="YB">YB</option>
+          </select>
+        </div>
+    
+        <div class="filter-item">
+          <label for="week1">1주차 금잔디조</label>
+          <input type="text" id="week1" placeholder="1주차 금잔디조를 입력하세요 (1~9)">
+        </div>
+
+        <div class="filter-item">
+          <label for="week2">2주차 금잔디조</label>
+          <input type="text" id="week2" placeholder="2주차 금잔디조를 입력하세요 (1~9)">
+        </div>
+      </div>
+    
+      <div class="button-group">
+        <button type="reset" id="reset-button">초기화</button>
+        <button type="submit" id="search-button">검색</button>

```

Comment:
p3: 해당 부분은 폼을 제출하는 것은 아니니 `type="button"`으로 수정해주는 것이 어떨까요?
```suggestion
        <button type="button" id="search-button">검색</button>
```

버튼 type 관련된 아티클 첨부하고 갑니다! 어려운 내용 아니어서 한번 쑥 읽기 좋아요.
https://nykim.work/96

Reply:
넵 감사합니다! 앞으로 type 지정할 때 놓치지 않게 조심해야겠어요